src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.Desktop.cs

```diff
@@ -63,13 +68,9 @@ public bool IsHostAssembly(Assembly assembly)
             return false;
         }
 
-        private partial Assembly? Load(AssemblyName assemblyName, string assemblyOriginalPath)
+        private partial Assembly? Load(AssemblyName assemblyName, string assemblyRealPath)
```

Comment:
doc'ing (at some level) what 'real' means would be helpful.  I genuinely am not sure what that means.